examples/let-clause/let_clause.bal

```diff
@@ -1,26 +1,33 @@
 import ballerina/io;
 
-type Person record {
- string first;
- string last;
- int yearOfBirth;
-};
+type Person record {|
```

Comment:
Since the record contains details of marks, shall we change `Person` to `Student`?